db/migrate/20250108231212_validate_foreign_keyfor_state_file_archived_intake_access_logs.rb

```diff
@@ -0,0 +1,5 @@
+class ValidateForeignKeyforStateFileArchivedIntakeAccessLogs < ActiveRecord::Migration[7.1]
+  def change
+    validate_foreign_key :state_file_archived_intake_access_logs, :state_file_archived_intake_requests
```

Comment:
can you explain why we have two migrations, with one saying `validate: false` for this column but then this one validates?

Reply:
this was a suggestion from the [strong_migrations ](https://github.com/ankane/strong_migrations) gem. i've attached a screenshot of the general rule. the idea is that adding the foreign key & validating the migration lock both tables. 

granted, there is no data in these tables, so locking them shouldn't be an issue, but strong migrations was loud enough that the only work around was to create two migrations.
<img width="826" alt="Screenshot 2025-01-13 at 3 53 00 PM" src="https://github.com/user-attachments/assets/d4b19c14-049b-4631-a948-9a5abdf4b75a" />